recipes/numbalsoda_recipe/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "numbalsoda" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Nicholaswogan/numblsoda/archive/refs/tags/v0.2.1.tar.gz
+  sha256: 6b0ee980d3b39882bf952488b452663b206da5fff04b27bdfb5fad563fdd89ce
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install --no-deps --no-build-isolation --ignore-installed . -v"
```

The other flags are added automatically via environment variables:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```